### PR TITLE
Image replacement

### PR DESCRIPTION
**Pull request by ShadabFaiz**
**Summary:**
Replaced all the jpg and png images with their webp format.
Previous Assets Size: 34.6 MB
New Asset Size: 9.6MB

---
**Fixes issue (include link):**

---
**Mockups, screenshots, evidence:**

### CODE REVIEW

**Comment by TobiTenno** [comment]
does android support webp for web icons instead of pngs?

File: src/assets/less/common.less [hunk 1]
```diff
@@ -94,10 +94,10 @@ img {
   margin-top: 20px;
 }
 
-.modal-body {
```
Comment (TobiTenno) [inline src/assets/less/common.less]: did you mean to add a bunch of css changes as well?

**Comment by SlayerOrnstein** [comment]
Yes, just checked the changes on Firefox and chromium. Images load perfectly.

File: src/components/modalDialogs/Settings.vue [hunk 2]
```diff
@@ -30,7 +30,7 @@
             v-model="theme"
             name="theme radios"
             v-on:change="updateTheme"
-            class="settings-group"
+            class="settings-group themeTabWrapper"
```
Comment (TobiTenno) [inline src/components/modalDialogs/Settings.vue]: dunno why this is duplicated in here.... looks like you're not branched off of the latest of master.

also, i thought the classes were dash separated... must've missed this one.

Comment (TobiTenno) [reply src/components/modalDialogs/Settings.vue]: yeah, looks like you need to rebase

**Comment by TobiTenno** [comment]
@SlayerOrnstein i mean for the pwa icons. I'm not seeing manifest changes, and i'm worried that the apple specific icons may not work either.

**Comment by SlayerOrnstein** [comment]
I can't really speak for the manifest as I'm not 100% sure how that works, but for the apple specific ones I sorta can. According to webp's website most browser except safari fully support webp. So long as PWA is running on a supported it should be fine.

https://developers.google.com/speed/webp/faq#which_web_browsers_natively_support_webp

**Comment by TobiTenno** [comment]
Yeah, I know android browsers themselves support webp, I meant specifically for the manifest stuff. This pr is based on the discussion from #408

**Comment by SlayerOrnstein** [comment]
@TobiTenno okay I think I see what you mean, the manifest needs to add the new webp images and the apple specific ones as far as the docs go have to be PNG.

**Comment by TobiTenno** [comment]
so, to summarize... @ShadabFaiz if you could update the manifest for the android icon changes, and revert the apple ones, that'd be swanky.

**Comment by ShadabFaiz** [comment]
> so, to summarize... @ShadabFaiz if you could update the manifest for the android icon changes, and revert the apple ones, that'd be swanky.

Sure. Also, safari may not officially support webp images, but I did check it on safari and they were working without any issue.
Also, there are few images under **public/img/icon** directory such as  **msapplication-icon-144x144.png**, but I'm not able to see where it is being used.

**Comment by TobiTenno** [comment]
They were part of the original Vue bundle that we got and then we swapped them for our logo. I'd say keep those as they were until we know for sure/can find where they're supposed to be. Webpack & Vue do a lot of automagic when they build the distributable bundle.

**Comment by SlayerOrnstein** [comment]
I believe msapplication are used for IE 10+, EdgeHTML uses mstile, and Edge chromium uses the regular chrome ones.

**Review by TobiTenno — APPROVED** [review]
LGTM

**Comment by ShadabFaiz** [comment]
Asking just because of curiosity. Is there anything still wrong with this PR or are we waiting for @SlayerOrnstein to give a pass?

**Comment by TobiTenno** [comment]
nope, i just need to test it a few places to make sure it's good all the places i can think of

**Comment by ShadabFaiz** [comment]
Ok

**Pull request merged**